pkg/cmd/roachtest/registry/testdata/filter/errors

```diff
@@ -1,9 +1,5 @@
 # Invalid filters.
 
-filter cloud=badcloud
-----
-error: invalid cloud "badcloud"; valid clouds are local,gce,aws,azure
```

Comment:
Not quite sure why this test case was removed?

Reply:
TFTR! This code that this tests now takes a `spec.Cloud`. It's no longer technically possible to pass an arbitrary cloud string.